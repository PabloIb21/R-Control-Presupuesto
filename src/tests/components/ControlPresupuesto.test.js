import React from 'react';
import { shallow } from 'enzyme';
import ControlPresupuesto from '../../components/ControlPresupuesto';

describe('<ControlPresupuesto />', () => {
  test('mount', () => {
    const wrapper = shallow(
      <ControlPresupuesto presupuesto={1000} restante={200} />
    );
  
    expect(wrapper).toMatchSnapshot();
  });
});