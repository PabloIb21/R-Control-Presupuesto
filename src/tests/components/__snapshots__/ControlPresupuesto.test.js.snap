// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<Fragment>
  <div
    className="alert alert-primary"
  >
    Presupuesto: $ 
    1000
  </div>
  <div
    className="alert alert-danger"
  >
    Restante: 
    200
  </div>
</Fragment>
`;

exports[`<ControlPresupuesto /> mount 1`] = `
<Fragment>
  <div
    className="alert alert-primary"
  >
    Presupuesto: $ 
    1000
  </div>
  <div
    className="alert alert-danger"
  >
    Restante: 
    200
  </div>
</Fragment>
`;

exports[`<ControlPresupuesto /> mount component 1`] = `
<Fragment>
  <div
    className="alert alert-primary"
  >
    Presupuesto: $ 
    1000
  </div>
  <div
    className="alert alert-danger"
  >
    Restante: 
    200
  </div>
</Fragment>
`;
